bootstrap-salt.sh

```diff
@@ -2663,6 +2669,11 @@ install_debian_deps() {
         __PIP_PACKAGES="${__PIP_PACKAGES} 'requests>=$_PY_REQUESTS_MIN_VERSION'"
     fi
 
+    if [ "$DISTRO_MAJOR_VERSION" -gt 8 ]; then
```

Comment:
Correct, it must be "-ge" instead of "-gt". My bad, sorry about that.

My intention was to install python-systemd on Debian 8 and higher, in case a installation from git is chosen. I had problems on Debian 8, when I chose installation from git and the python-systemd package was not installed. That was the reason for me to send the PR. 

If you think that the PR is unnecessary, I can revoke/delete it, then test further and come back with another PR if the git installation on Debian 8 still does not work for me without the systemd-python package.